.travis.yml

```diff
@@ -54,42 +73,14 @@ cache:
 
 matrix:
   include:
-    # Test different versions of SQLAlchemy
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.6.0 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.6.8 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.7.0 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.7.4 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.7.8 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=latest TESTS=trial
-
-    # add extra tests in separate jobs
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=coverage
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=lint
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=js
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=docs
-
+    - python: "2.6"
+      env: TWISTED=14.0.2 TESTS=worker-slave
```

Comment:
On the second though, I think better would be `workers`. And instead of `trial_worker` use `only_worker`.